### DISCUSSION TITLE
Confusion regarding initial score

### DISCUSSION THREAD

**Discussion by DelusionArun** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#9] [Q&A] [answered]
I am confused about what the following documentation line refers to:
<li>The initial score of any game board is {@code gameBoardSize}.</li>

What does gameBoardSize refer to?

**Comment by Derppening** [comment, marked as answer]
`gameBoardSize` is the size of the game board, i.e. `number of rows * number of columns`.
